sdks/python/apache_beam/runners/dataflow/internal/clients/cloudbuild/__init__.py

```diff
@@ -29,5 +29,3 @@
   from apache_beam.runners.dataflow.internal.clients.cloudbuild.cloudbuild_v1_messages import *
 except ImportError:
   pass
-
```

Comment:
these are autogenerated clients, why do we need these changes?

Reply:
Removed this changes. They were some errors saying `__path__ ` was not defined initially but they are solved now